definitions/procedures/pulp/remove.rb

```diff
@@ -70,7 +81,7 @@ def ask_to_proceed(rm_cmds)
     end
 
     def run
-      rm_cmds = data_dir_removal_cmds
+      rm_cmds = data_dir_removal_cmds(pulp_data_dirs)
```

Comment:
you now call `data_dir_removal_cmds` inside `delete_pulp_data`, so this is not really needed anymore, if you also move the `if rm_cmds.any` *inside* `delete_pulp_data` on line 100?

Reply:
Moved it to `ask_to_proceed` as its not being shared with `delete_pulp_data` anymore.